features/nationaliteit.feature

```diff
@@ -1,59 +1,301 @@
+      | 1      | 4         | 0100                  | 311                   |                          |                                    |                 | 20200727                        |
+      | 1      | 54        | 0100                  | 311                   |                          |                                    | O               | 20200713                        |
+      Als personen wordt gezocht met de volgende parameters
+      | naam                | waarde                              |
+      | type                | RaadpleegMetBurgerservicenummer     |
+      | burgerservicenummer | 555550003                           |
+      | fields              | burgerservicenummer,nationaliteiten |
+      Dan heeft de persoon met burgerservicenummer '555550003' een nationaliteit met alleen de volgende gegevens
+      | naam                  | waarde   |
+      | nationaliteit         | 0100     |
+      | redenOpname           | 311      |
+      | datumIngangGeldigheid | 20200727 |
+
+  @proxy
+  Rule: de nationaliteit wordt gevuld op basis van het voorkomen van nationaliteit of aanduidingBijzonderNederlanderschap
+    - wanneer nationaliteit voorkomt met een waarde ongelijk aan "0000" (onbekend), dan wordt type opgenomen met de waarde "Nationaliteit"
+    - wanneer nationaliteit voorkomt met een waarde gelijk aan "0000" (onbekend), dan wordt type opgenomen met de waarde "NationaliteitOnbekend"
+    - wanneer aanduidingBijzonderNederlanderschap voorkomt met een waarde gelijk aan "B", dan wordt type opgenomen met de waarde "BehandeldAlsNederlander"
+    - wanneer aanduidingBijzonderNederlanderschap voorkomt met een waarde gelijk aan "V", dan wordt type opgenomen met de waarde "VastgesteldNietNederlander"
+
+    @proxy
+    Scenario: de persoon heeft de Nederlandse nationaliteit
+      Gegeven het systeem heeft een persoon met de volgende gegevens
+      | naam                | waarde    |
+      | burgerservicenummer | 000009830 |
+      En de persoon met burgerservicenummer '000009830' heeft een nationaliteit met de volgende gegevens
+      | naam                  | waarde   |
+      | nationaliteit         | 0001     |
+      | redenOpname           | 001      |
+      | datumIngangGeldigheid | 19750707 |
+      Als personen wordt gezocht met de volgende parameters
+      | naam                | waarde                              |
+      | type                | RaadpleegMetBurgerservicenummer     |
+      | burgerservicenummer | 000009830                           |
+      | fields              | burgerservicenummer,nationaliteiten |
+      Dan heeft de persoon met burgerservicenummer '000009830' een nationaliteit met alleen de volgende gegevens
+      | naam                  | waarde        |
+      | type                  | Nationaliteit |
+      | nationaliteit         | 0100          |
```

Comment:
kloppen deze waarden? Ze komen niet overeen met de waarden in de 'gegeven' stap. Ook is datumIngangGeldigheid niet van de polymorfe type.

Reply:
klopte inderdaad niks van. gecorrigeerd